controllers/admin/AdminPreferencesController.php

```diff
@@ -42,51 +42,51 @@ public function __construct()
             $round_mode = array(
                 array(
                     'value' => PS_ROUND_HALF_UP,
-                    'name' => $this->l('Round up away from zero, when it is half way there (recommended)')
+                    'name' => $this->trans('Round up away from zero, when it is half way there (recommended)', array(), 'Admin.ShopParameters.Feature')
                 ),
                 array(
                     'value' => PS_ROUND_HALF_DOWN,
-                    'name' => $this->l('Round down towards zero, when it is half way there')
+                    'name' => $this->trans('Round down towards zero, when it is half way there', array(), 'Admin.ShopParameters.Feature')
                 ),
                 array(
                     'value' => PS_ROUND_HALF_EVEN,
-                    'name' => $this->l('Round towards the next even value')
+                    'name' => $this->trans('Round towards the next even value', array(), 'Admin.ShopParameters.Feature')
                 ),
                 array(
                     'value' => PS_ROUND_HALF_ODD,
-                    'name' => $this->l('Round towards the next odd value')
+                    'name' => $this->trans('Round towards the next odd value', array(), 'Admin.ShopParameters.Feature')
                 ),
                 array(
                     'value' => PS_ROUND_UP,
-                    'name' => $this->l('Round up to the nearest value')
+                    'name' => $this->trans('Round up to the nearest value', array(), 'Admin.ShopParameters.Feature')
                 ),
                 array(
                     'value' => PS_ROUND_DOWN,
-                    'name' => $this->l('Round down to the nearest value')
+                    'name' => $this->trans('Round down to the nearest value', array(), 'Admin.ShopParameters.Feature')
```

Comment:
This need a semi-colon:

```php
array(
    'foo' => 'bar',
    'bar' => 'baz',
)
```

Why ? *Because* the next time you add an array entry, github will only highlight the last entry and not the line with 
the semi colon.

I will clean this, this comment is just for information purposes :)